Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -479,71 +478,70 @@ NumberCombinedWithDurationUnit: !nestedRegex
   def: \b(?<num>\d+(\.\d*)?)(-)?{DurationUnitRegex}
   references: [ DurationUnitRegex ]
 AnUnitRegex: !nestedRegex
-  def: \b((?<half>half\s+)?(an|a)|another)\s+{DurationUnitRegex}
+  def: \b(?<half>(een\s(half|halve)))\s+{DurationUnitRegex}
   references: [ DurationUnitRegex ]
 DuringRegex: !simpleRegex
-  def: \b(for|during)\s+the\s+(?<unit>jaar|maand|week|dag)\b
+  def: \b(voor\s+een|gedurende\s+(het|de))\s+(?<unit>jaar|maand|week|dag)\b
 AllRegex: !simpleRegex
-  def: \b(?<all>(all|full|whole)(\s+|-)(?<unit>jaar|maand|week|dag))\b
+  def: \b(?<all>((ge)?hele|volledige|ganse)(\s+|-)(?<unit>jaar|maand|week|dag))\b
```

Comment:
Suggestion: should `heel jaar` and `volledig jaar` also be supported in this one?

Reply:
It is a somewhat tricky area, since often it depends on gender of the following words, whether adjectives need an extra /e/.

Reply:
Understand. Guess this is also not an important one, so would just skip this one for now. 
You can always add testcases for those and disable them + add a comment.